scripts/eurostat/regional_statistics_by_nuts/education_enrollment/Eurostats_NUTS2_Enrollment.tmcf

```diff
@@ -1,27 +1,31 @@
 
-Node: E:EurostatsNUTS2_Enrollment->E0
-typeOf: dcs:StatVarObservation
-variableMeasured: dcs:Count_Person_25To64Years_EnrolledInEducationOrTraining_Female_AsAFractionOfCount_Person_25To64Years_Female
-observationAbout: C:EurostatsNUTS2_Enrollment->GeoId
-observationDate: C:EurostatsNUTS2_Enrollment->Date
-value: C:EurostatsNUTS2_Enrollment->Count_Person_25To64Years_EnrolledInEducationOrTraining_Female_AsAFractionOfCount_Person_25To64Years_Female
-scalingFactor: 100
-measurementMethod: dcs:EurostatRegionalStatistics
-
-Node: E:EurostatsNUTS2_Enrollment->E1
-typeOf: dcs:StatVarObservation
-variableMeasured: dcs:Count_Person_25To64Years_EnrolledInEducationOrTraining_Male_AsAFractionOfCount_Person_25To64Years_Male
-observationAbout: C:EurostatsNUTS2_Enrollment->GeoId
-observationDate: C:EurostatsNUTS2_Enrollment->Date
-value: C:EurostatsNUTS2_Enrollment->Count_Person_25To64Years_EnrolledInEducationOrTraining_Male_AsAFractionOfCount_Person_25To64Years_Male
-scalingFactor: 100
-measurementMethod: dcs:EurostatRegionalStatistics
-
-Node: E:EurostatsNUTS2_Enrollment->E2
-typeOf: dcs:StatVarObservation
-variableMeasured: dcs:Count_Person_25To64Years_EnrolledInEducationOrTraining_AsAFractionOfCount_Person_25To64Years
-observationAbout: C:EurostatsNUTS2_Enrollment->GeoId
-observationDate: C:EurostatsNUTS2_Enrollment->Date
-value: C:EurostatsNUTS2_Enrollment->Count_Person_25To64Years_EnrolledInEducationOrTraining_AsAFractionOfCount_Person_25To64Years
-scalingFactor: 100
-measurementMethod: dcs:EurostatRegionalStatistics
+        Node: E:EurostatsNUTS2_Enrollment->E0
```

Comment:
Pls remove the leading spaces at the start of files below.

Reply:
removed